css/style.scss

```diff
@@ -0,0 +1,359 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap');
+$light-gry:#e9e9e9;
+$cgreen-color:#10c9c3;
+$white-color:#fff;
+$dark-brown:#191919;
+$gry:#838383;
+$section-gap:100px 0;
+$container:1170px;
+$center:0 auto;
```

Comment:
Why these variables are still defined here ? 
We have defined a separate file for variables.